### ISSUE TITLE
Available plugins list - Plugins list is randomly empty

### ISSUE THREAD

**Issue by Gaetanbrl** [georchestra/mapstore2-georchestra#313]
### Issue

After upload a little custom plugin inside new saved context, available plugins list seems empty :

![image](https://user-images.githubusercontent.com/16317988/104184667-172f7f80-5414-11eb-8941-dea42eefcc74.png)

Steps to reproduce :

* Create contexte and upload a new plugin with Annotation plugin ==> OK
* Open the new context ==> OK
* Create new context and try to upload a random plugin from list ==> List is empty ==> KO
* Retry ==> Work (sometime) ==> OK / KO

Sometime it's KO result : 

![image](https://user-images.githubusercontent.com/16317988/104185911-ec462b00-5415-11eb-8d1c-c6eb9ca1bada.png)

If i refresh the page, it's OK :

![image](https://user-images.githubusercontent.com/16317988/104185965-008a2800-5416-11eb-9aae-6678af27d5c4.png)

* Try to change the created context to add Measure plugin ==> KO / OK again

**Logs**

No logs from console or server side.

**File system**

I use the geOrchestra datadir (`/etc/georchestra/mapstore`) ang could access to MS2 files.

**Comment by pierrejego** [comment]
Hi,
I have reproduced this issue on a fresh instance : 
 
On console we can see :
 Failed to load resource: the server responded with a status of 404 ()      /mapstore/rest/config/load/extensions.json:1

**Comment by pierrejego** [comment]
@catmorales Do you have the same on your servers ?

**Comment by catmorales** [comment]
@pierrejego Yes i have the same error in the console but it don't seem to be a problem.
I had the same issue as you with sometimes the plugin list empty and with  a new deployment of mapstore iit was solved 
cf https://github.com/georchestra/mapstore2-georchestra/issues/263

I think it's linked with the tutoriel ogf the context manager :
![image](https://user-images.githubusercontent.com/4659829/106251933-e9b94300-6215-11eb-9382-0c099e852abb.png)

**Comment by Gaetanbrl** [comment]
> with a new deployment of mapstore iit was solved

You mean you realized a  total clean + install or just upgrade mapstore (`pull master + build + war deploy`) ?

Because i had upgraded mapstore  yesterday (_total clean git folder + clone master recursive + install + build + deploy war_) and this solution don't fix this issue (in my case).

I **ALWAYS** need to refresh contexte creator page to get full plugins list.

**Comment by MaelREBOUX** [comment]
Hello @Gaetanbrl  is this behaviour still existing ?

**Comment by Gaetanbrl** [comment]
Hi,
Need to upgrade local MapStore to try again.

But @isanchez-c2c reproduced this behavior during the geOcom.
